groovy/postInit/electronics/ElectronicCircuits.groovy

```diff
@@ -10,38 +12,60 @@ mods.gregtech.assembler.removeByInput(30, [metaitem('wireFineCopper') * 4, metai
 mods.gregtech.assembler.removeByInput(30, [metaitem('wireFineAnnealedCopper') * 4, metaitem('dustSmallGalliumArsenide')], [fluid('plastic') * 144])
 // Diode * 2
 mods.gregtech.assembler.removeByInput(30, [metaitem('wireFineCopper') * 4, metaitem('dustSmallGalliumArsenide')], [fluid('plastic') * 144])
+// Resistor * 2
+mods.gregtech.assembler.removeByInput(6, [metaitem('dustCoal'), metaitem('wireFineCopper') * 4], [fluid('glue') * 100])
+// Resistor * 2
+mods.gregtech.assembler.removeByInput(6, [metaitem('dustCharcoal'), metaitem('wireFineCopper') * 4], [fluid('glue') * 100])
+// Resistor * 2
+mods.gregtech.assembler.removeByInput(6, [metaitem('dustCarbon'), metaitem('wireFineCopper') * 4], [fluid('glue') * 100])
+// Resistor * 4
+mods.gregtech.assembler.removeByInput(6, [metaitem('dustCoal'), metaitem('wireFineAnnealedCopper') * 4], [fluid('glue') * 100])
+// Resistor * 4
+mods.gregtech.assembler.removeByInput(6, [metaitem('dustCharcoal'), metaitem('wireFineAnnealedCopper') * 4], [fluid('glue') * 100])
+// Resistor * 4
+mods.gregtech.assembler.removeByInput(6, [metaitem('dustCarbon'), metaitem('wireFineAnnealedCopper') * 4], [fluid('glue') * 100])
+
+crafting.removeByOutput(metaitem('component.resistor')) 
+
+carbons = new ItemStack[]{
+        metaitem('dustCoal'),
+        metaitem('dustCharcoal'),
+        metaitem('dustCarbon'),
+        metaitem('dustCoal'),
+        metaitem('dustHighPurityCarbon'),
+        metaitem('dustAnthracite'),
+        metaitem('dustCoal')
```

Comment:
me when I am a duplicate

Reply:
actually I am a triplicate and I also don't include dustCoke

Reply:
oops
i am stupd